packages/coolprop/meta.yaml

```diff
@@ -1,25 +1,20 @@
 package:
   name: coolprop
-  version: 6.6.0
-  # Problem with our added pyproject.toml/setup.py
+  version: 7.2.0
+  # Now uses upstream scikit-build-core-based build; local pyproject.toml/setup.py no longer needed
   pinned: true
```

Comment:
```suggestion
```

This PR updates the version and removed the `setup.py` so the `pinned` and the commend is not needed anymore.